asian-myrmecology.csl

```diff
@@ -0,0 +1,215 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" demote-non-dropping-particle="sort-only" version="1.0" class="in-text" default-locale="en-GB">
+  <info>
+    <title>Asian Myrmecology</title>
+    <id>http://www.zotero.org/styles/asian-myrmecology</id>
+    <link href="http://www.zotero.org/styles/asian-myrmecology" rel="self"/>
+    <link href="http://www.zotero.org/styles/restoration-ecology" rel="template"/>
+    <link href="http://www.asian-myrmecology.org/guidelines.html#:~:text=of%20the%20manuscript.-,References,-Authors%20are%20encouraged" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="biology"/>
+    <issn>1985-1944</issn>
+    <eissn>2462-2362</eissn>
+    <updated>2021-11-10T16:08:06+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="accessed">downloaded from</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor">
+      <name initialize-with="" sort-separator=", " name-as-sort-order="all" delimiter=", " and="symbol">
+        <name-part name="family" text-case="capitalize-first"/>
+        <name-part name="given" text-case="capitalize-first"/>
+      </name>
+      <label form="short" prefix=", "/>
+    </names>
+  </macro>
+  <macro name="series-editor">
+    <names variable="original-author">
+      <label text-case="capitalize-first" form="short" suffix=" "/>
+      <name delimiter=", " and="symbol"/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name initialize-with="" sort-separator=" " name-as-sort-order="all" delimiter-precedes-last="always" et-al-min="4"/>
+      <label prefix=", " form="short"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+      </substitute>
```

Comment:
Could you add title to substitute here and in author-short? See webpage on preview, which doesn't look right. (Otherwise, the style guide is a hot mess, so this looks like the most reasonable implementation...)

Reply:
Done. Thanks for reviewing. :)